test/test_images/probe_helper/probe_helper.yaml

```diff
@@ -39,6 +39,8 @@ spec:
               value: "http://default-brokercell-ingress.cloud-run-events.svc.cluster.local/cloud-run-events-probe/default"
             - name: CLOUDPUBSUBSOURCE_TOPIC_ID
               value: "cloudpubsubsource-topic"
+            - name: CLOUDSTORAGESOURCE_BUCKET_ID
+              value: "project-id-cloudstoragesource-bucket"
```

Comment:
Since Cloud Storage bucket names are global (not project-specific), I felt it made sense to include the project name within. It's not really important in the default config anyway, so I can remove it, but that's the reason for the scheme as it stands.

Reply:
ACK